test/functional/system/snapshotting.js

```diff
@@ -51,18 +54,32 @@ describe('snapshotting', () => {
 
 		before(() => {
 			// Forge 201 blocks to reach height 202 (genesis block is already tyhere)
-			return Promise.mapSeries([...Array(201)], () => {
+			return Promise.mapSeries([...Array(99)], () => {
```

Comment:
Its a bit confusing you are forging `99` blocks here and comment above states `201` blocks. After `genesisBlock` at height `1` and then rest of `99` blocks we will have 100 blocks, how you calculated those to 201?

Reply:
If we see steps just below the total blocks become 

```
1 + 99 + 101 + 101 = 302
```

Reply:
Yeah, you're and @MaciejBaj have right, I will update the comment.